pip/requirements.txt

```diff
@@ -281,7 +281,7 @@ tenacity==8.0.1
 #NO_AUTO_UPDATE:3: Force to use tensorflow 2.6.0; this should match the version in tensorflow-sources.spec
 tensorboard==2.6.0
 tensorflow==2.6.0
-tensorflow-estimator==2.6.0
+tensorflow-estimator==2.8.0
```

Comment:
I think we should keep this one at 2.6.0 as well, to match tensorflow. But could be wrong. @smuzaffar ?

Reply:
Let's try the test for 2.6.0 (tensorflow latest version is also 2.8.0)

Reply:
right, we need to keep it to 2.6.0 (to match the version in `tensorflow-sources`)